centreon/www/front_src/src/Resources/Filter/criteriasNewInterface/criterias.style.ts

```diff
@@ -29,7 +29,8 @@ export const useStyles = makeStyles()((theme) => ({
     paddingLeft: theme.spacing(1)
   },
   inputInformation: {
-    backgroundColor: theme.palette.background.default
+    backgroundColor: theme.palette.background.default,
+    minWidth: theme.spacing(35)
```

Comment:
What is this for?

Reply:
update width of information input to mach the mockup